modules/subscriber/variables.tf

```diff
@@ -104,3 +104,10 @@ variable "lambda_env_vars" {
   nullable    = false
   default     = {}
 }
+
+variable "debug_endpoint" {
+  description = "Endpoint to send debugging telemetry to. Ask your Observe POC for further instructions"
+  type        = string
+  nullable    = true
```

Comment:
If you pass `null` your terraform will break because environment variables must be strings.